tests/ctst/steps/utils/kubernetes.ts

```diff
@@ -63,6 +63,22 @@ export function createKubeCustomObjectClient(world: Zenko) {
     return KubernetesHelper.customObject as CustomObjectsApi;
 }
 
+export async function deletePodsByLabel(world: Zenko, label: string, namespace = 'default') {
+    const coreClient = createKubeCoreClient(world);
+
+    try {
+        await coreClient.deleteCollectionNamespacedPod(
+            namespace, undefined, undefined, undefined, undefined, undefined, label);
+    } catch (err) {
+        world.logger.error('Error deleting pods by label', {
```

Comment:
```suggestion
        world.logger.debug('Error deleting pods by label', {
```

Error logs will dump all the logs in-flight for the "request", here the scenario; but we already do it at the end of any scenario if the result is "failed"

Reply:
(I know we have "error" in other places)